docs/src/modules/components/examples/coreExamples.ts

```diff
@@ -109,7 +108,7 @@ const coreExamples = [
     src: '/static/toolpad/docs/core/tutorial-1.png',
     srcDark: '/static/toolpad/docs/core/tutorial-1.png', // TODO Fix
     href: 'https://mui.com/toolpad/core/introduction/tutorial/',
-    source: 'https://github.com/mui/toolpad/tree/master/examples/core/tutorial/',
+    source: 'https://github.com/mui/toolpad/tree/master/examples/core/tutorial',
```

Comment:
GitHub supports both, they have no care for forcing one over the other, it's so cheap, lol, not even a canonical link. 
In any case, here, we can use what their links default to.

x same as for the other diffs.